README.md

```diff
@@ -477,6 +477,31 @@ module.exports = {
 }
 ```
 
+#### Default Babel configuration
+Next.js uses Babel6 out of the box, which doesn't support default `module.exports`.
+##### Invalid:
+```js
+import React from 'react':
```

Comment:
Delete this line. It's not required with Next.
And without that, example looks clean.